src/reducer/admin/notification.js

```diff
@@ -6,12 +6,17 @@ import {
 const defaultState = {
     text: '',
     type: 'info', // one of 'info', 'confirm', 'warning'
+    autoHideDuration: undefined, // undefined means use global duration
```

Comment:
This is where you should put the default duration of 4000

Reply:
if you put it here it will override the global one set in prop though

Reply:
gotcha

Reply:
Replace "undefined  means use global duration" by "use default duration from Notification"